src/main/java/org/broadinstitute/hellbender/engine/dataflow/datasources/ReferenceDataflowSource.java

```diff
@@ -0,0 +1,40 @@
+package org.broadinstitute.hellbender.engine.dataflow.datasources;
+
+import com.google.cloud.dataflow.sdk.options.PipelineOptions;
+import com.google.cloud.dataflow.sdk.transforms.SerializableFunction;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.dataflow.BucketUtils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+import org.broadinstitute.hellbender.utils.reference.ReferenceBases;
+
+import java.io.IOException;
+import java.io.Serializable;
+
+public class ReferenceDataflowSource implements ReferenceSource, Serializable {
```

Comment:
Agreed - I was getting this out quickly for people to have a look at. I'll add a class comment and make it final.